pkg/crc/services/dns/template.go

```diff
@@ -59,13 +62,27 @@ func createDnsmasqDNSConfig(serviceConfig services.ServicePostStartConfig) error
 	if err != nil {
 		return err
 	}
+
+	dnsHostConfig, err := createDnsConfigFile(dnsmasqConfFileValues, dnsmasqHostsTemplate)
+	if err != nil {
+		return err
+	}
+
+	encodeddnsConfig = base64.StdEncoding.EncodeToString([]byte(dnsHostConfig))
+	_, err = serviceConfig.SSHRunner.Run(
+		fmt.Sprintf("echo '%s' | openssl enc -base64 -d | sudo tee /var/srv/hosts.openshift > /dev/null",
+			encodeddnsConfig))
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
-func createDnsConfigFile(values dnsmasqConfFileValues) (string, error) {
+func createDnsConfigFile(values dnsmasqConfFileValues, temp string) (string, error) {
```

Comment:
`template` instead of `temp`, I read it as `temporary`

Reply:
@cfergeau `template` is name of imported module so I would go for `tmpl` :)